### PR TITLE
Cleaning up

### PR DESCRIPTION
**Pull request by Melancholick**

### CODE REVIEW

File: assets/themes/bootstrap/resources/bootstrap/js/render_jobad_list_filter.js [hunk 1]
```diff
@@ -27,120 +27,92 @@ function tableLoad() {
...
-                filterTag.remove(filterName.indexOf(filterTag));
-            }
-//This part will be rewriten latter, but so far its work. Its check if the element in json_data has all the tags that is in filtering query array(they all return true).
+        /**
+         * Default filter for table.
+         */
+        var filterTag = ['REMOTE1_100'];
+        $(".loader").hide();
+        /**
+         * Function that setups table. Look wenzhixin documentation for details.
```
Comment (stargazer33) [inline assets/themes/bootstrap/resources/bootstrap/js/render_jobad_list_filter.js:27]: Insert a direct link to documentation

File: assets/themes/bootstrap/resources/bootstrap/js/render_jobad_list_filter.js [hunk 2]
```diff
@@ -27,120 +27,92 @@ function tableLoad() {
...
+        console.log(err);
+    }
 });
-
-//Formatter for detailed view(Full view of entry row). 
+/**
+ * Formating data from json_data for detailed view of every row.
+ * for every key:value dictionaries of row it decorates and pushes strings to html.
+ */
 function detailFormatter(index, row) {
```
Comment (stargazer33) [inline assets/themes/bootstrap/resources/bootstrap/js/render_jobad_list_filter.js]: describe every argument. 
What is the type of "row" ?
describe the result

It seems "index" unused - drop it ?

File: assets/themes/bootstrap/resources/bootstrap/js/render_jobad_list_filter.js [hunk 3]
```diff
@@ -27,120 +27,92 @@ function tableLoad() {
...
-            }
+
                 return item.tags.some(hasTag);
             };
+            /**
+             * Loading table with filtered by default data.
+             */
+            tableLoad();
+        });
+        /**
```
Comment (stargazer33) [inline assets/themes/bootstrap/resources/bootstrap/js/render_jobad_list_filter.js]: Drop lines 84 - 92. 

It seems, there is no useful functionality there

File: assets/themes/bootstrap/resources/bootstrap/js/render_jobad_list_filter.js [hunk 4]
```diff
@@ -1,11 +1,8 @@
...
-  this.length = from < 0 ? this.length + from : from;
-  return this.push.apply(this, rest);
-};
-
+/**
+ * Decorate tags from given array into html string, set of labeled tags.
+ * @param {array} tags; ex: row.tagsNames1, row.tags;
+ * @return {string} rtTags, a sum decorated tags;
+ */
 function tagsDeco(tags) {
```
Comment (stargazer33) [inline assets/themes/bootstrap/resources/bootstrap/js/render_jobad_list_filter.js]: move this function to the end of file

File: assets/themes/bootstrap/resources/bootstrap/js/render_jobad_list_filter.js [hunk 5]
```diff
@@ -152,12 +124,23 @@ function detailFormatter(index, row) {
     });
     return html.join('');
 }
-//Formatter for published column. It take a value and split it by ','. ANd take only first part with number of day and month.
+/**
+ * Formater for published column.
+ * @param {string} value; value of item.published for this row;
+ * @var {string} publishedDay; split from value only the part with day and month;
```
Comment (stargazer33) [inline assets/themes/bootstrap/resources/bootstrap/js/render_jobad_list_filter.js]: не надо описывать переменные. удали эту строку

File: assets/themes/bootstrap/resources/bootstrap/js/render_jobad_list_filter.js (hunk 2 above)
Comment (Melancholick) [reply assets/themes/bootstrap/resources/bootstrap/js/render_jobad_list_filter.js]: No, its needed. It passes data-index of current row, and then this function know which row data its working with.

File: assets/themes/bootstrap/resources/bootstrap/js/render_jobad_list_filter.js (hunk 1 above)
Comment (Melancholick) [reply assets/themes/bootstrap/resources/bootstrap/js/render_jobad_list_filter.js:27]: Added documentation link

**Pull request merged**